webapp/lib/controller.dart

```diff
@@ -202,7 +203,7 @@ void listenForNeedsReplyMetrics(String project) {
   );
 }
 
-void listenForDriverMetrics(String project, List<String> drivers) {
+void listenForDriverMetrics(String project, Map<String, List<String>> drivers) {
```

Comment:
I'd rather we keep this method self-contained - it doesn't need to know that there are other `drivers` for other projects. So keep this method signature as it was, and instead call it with `DRIVERS[selectedProject]`.

Reply:
Okay. Reverted it back